misc/testerina/modules/testerina-core/src/main/java/org/ballerinalang/testerina/natives/test/FunctionMock.java

```diff
@@ -43,14 +43,18 @@ public static ErrorValue thenReturn(ObjectValue caseObj) {
 
     public static Object mockHandler(ObjectValue mockFuncObj, ArrayValue args) {
```

Comment:
Seems `thenReturn` and `mockHandler` methods never used. Pls remove them if it is not used.

Reply:
In this case, `thenReturn` and `mockHandler` are being used by the java interop in mock.bal. These are required for the functionality to work.